app/styles/_profile.scss

```diff
@@ -0,0 +1,147 @@
+@import "app-variables";
+
+.mdl-route__profile {
+  font-family: $waldo-font;
+  margin: auto;
+
+  header {
+      background-color: $waldo-blue;
+      display: flex;
+      justify-content: space-around;
+      align-items: center;
+      width: 100%;
+      height:30%;
+      position: absolute;
+      top: 0;
+      z-index:5;
+  }
+
+  .div-outer-white {
+      background-color:white;
+      margin-bottom:10%;
+  }
+
+  .div-inner-grey {
+      background-color: #CDCDCD;
+      width:90%;
+      margin-left:5%;
+      position: absolute;
+  }
+
+  .div-content{
+    position:relative;
+    width:90%;
+    margin-left:5%;
+    z-index:10;
+    margin-top:60%;
+    margin-bottom:30%;
+  }
+
+  .profile-top{
+    background-color:white;
+    text-align:center;
+    height:100px;
+    margin-top:-20%;
+    margin-bottom:5%
+  }
+
+  .profile-top img {
+    overflow:visible;
+    margin-top:-20%;
+    width:100px;
+    height:100px;
+    border: 2px;
+    border-color: white;
+  }
+
+  .profile-top h1 {
+    margin-bottom:-10px;
+  }
+
+  .profile-buttons {
```

Comment:
The `flex` style provides a great way to even out these buttons. You can just add
```css
display: flex;
justify-content: space-evenly;
```
to `.profile-buttons` to space them apart and center them

Reply:
Here is a link to a helpful tutorial on flex: http://flexboxfroggy.com/#en
I have found flex remarkably easy, simple, and really good looking